odkx-src/xlsx-converter-reference.rst

```diff
@@ -392,7 +392,7 @@ The following prompt types are available in ODK-X Survey.
     - | Used to enter the information in a big text area or pragraphs.
     
 .. note::
-   if users anticipates for writing anything longer than 255 characters then the user need to change the model sheet and change the elementType column. As the documentation states: "For example, as shown in the datatypes XLSX, string variables' length can be adjusted from a default of 255 to other lengths with string(len)". For example, if you had a string prompt named long_data that you wanted to be 500 characters, you would add the following to your model worksheet.
+   if users anticipates for writing anything longer than 255 characters then the user need to change the model sheet and change the element-type column. As the documentation states: "For example, as shown in the datatypes XLSX, string variables' length can be adjusted from a default of 255 to other lengths with string(len)". For example, if you had a string prompt named long_data that you wanted to be 500 characters, you would add the following to your model worksheet.
```

Comment:
At least in the current documentation, the column is named elementType, not "element-type" so I think this change may be incorrect...

Reply:
yes, I have written "elementType" only but the build is not passing. That shows it as spelling error

Reply:
you can add the word to https://github.com/odk-x/docs/blob/master/odkx-src/spelling_wordlist.txt and the spell checker will recognize it (there is an issue already to document this that someone is working on https://github.com/odk-x/tool-suite-X/issues/247)

Reply:
@wbrunette I have added the word and made the PR for that https://github.com/odk-x/docs/pull/82 please review that